### DISCUSSION TITLE
saf-training-current/courses/beginner/02

### DISCUSSION THREAD

**Discussion by giscus** [mitre/saf-training#215] [Ideas]
# saf-training-current/courses/beginner/02

Check-InHave you used InSpec before? There is no right or wrong answer. If this is your first time using InSpec, welcome! If you have used it before, welcome back. If you want m...

https://mitre.github.io/saf-training-current/courses/beginner/02.html

**Comment by p-oneil** [comment]
Closing - no content, just a reaction
